src/ui/server/middleware/headers/security.ts

```diff
@@ -26,9 +26,10 @@ export const security = (req: Request, res: Response, next: () => void) => {
   res.setHeader('X-Frame-Options', 'deny');
   res.setHeader('X-XSS-Protection', '1; mode=block');
   res.setHeader('X-Content-Type-Options', 'nosniff');
+  // TODO: script-src nounce https://developers.google.com/tag-platform/tag-manager/web/csp
```

Comment:
Do we need to create a ticket for this and add the icket number here?

Reply:
I will create a ticket and will add number, will do as part of Cypress PR.